### PR TITLE
SLING-9007: Base the region calculation on the feature-origin and add…

### PR DESCRIPTION
**Pull request by karlpauls**
… a feature-origin to apiregions during merge

### CODE REVIEW

File: src/test/java/org/apache/sling/feature/extension/apiregions/APIRegionMergeHandlerTest.java [hunk 1]
```diff
@@ -144,103 +146,15 @@ public void testRegionExportsNoInheritance() throws Exception {
 
         Extension tgEx = tf.getExtensions().iterator().next();
 
-        String expectedJSON = "[{\"name\":\"global\",\"exports\":[\"a.b.c\",\"d.e.f\"]},"
-                + "{\"name\":\"deprecated\",\"exports\":[\"klm\",\"qrs\"]},"
-                + "{\"name\":\"internal\",\"exports\":[\"xyz\"]},"
-                + "{\"name\":\"forbidden\",\"exports\":[\"abc\",\"klm\"]}]";
+        String expectedJSON = "[{\"name\":\"global\",\"exports\":[\"a.b.c\",\"d.e.f\"],\"feature-origins\":\"y:s:2\"},"
```
Comment (bosschaert) [inline src/test/java/org/apache/sling/feature/extension/apiregions/APIRegionMergeHandlerTest.java]: should `feature-origins` not be a JSON Array?

**Comment by karlpauls** [comment]
@bosschaert, I guess so - I did it this way mostly for consistency with the metadata approach in the Artifact but I guess we can do it correctly here. Anyways, I changed it now  - hence, PTAL again :-)

**Comment by bosschaert** [comment]
Thanks @karlpauls ! LGTM!

**Comment by bosschaert** [comment]
In my recent PR https://github.com/apache/sling-org-apache-sling-feature-apiregions/pull/2 I'm also getting a 0% coverage similar to here. Seems like the Sonar coverage part is not properly configured...

**Pull request merged**